ComponentKit/TransactionalDataSources/Common/Internal/CKTransactionalComponentDataSourceChangesetModification.mm

```diff
@@ -56,6 +58,8 @@ - (CKTransactionalComponentDataSourceChange *)changeFromState:(CKTransactionalCo
   }];
 
   // Update items
+  // This structure is used for computing post-changeset-application index paths for updated items.
+  __block std::unordered_map<CKTransactionalComponentDataSourceItem *, NSIndexPath *, CK::HashFunctor<NSObject *>> itemToInitialIndexPathMap;
```

Comment:
Also looks like `CKFunctor<NSObject *>` is just casting the pointer, which is a fairly poor hash function…